Android/app/src/main/java/uncommon/common/activity/PlaceActivity.java

```diff
@@ -1,159 +1,197 @@
-        });
-
-        ApiInterface service = RetrofitInstance.getRetrofitInstance().create(ApiInterface.class);
-        Call<Reservation> request = service.getReservation("kahye5232@naver.com");
-        request.enqueue(new Callback<Reservation>() {
-            @Override
-            public void onResponse(Call<Reservation> call, Response<Reservation> response) {
-                //upcoming notification
-                if (response.body().isValid()) {
-                    Reservation res = response.body();
-
-                    //convert date format yyyy-MM-dd into dd-MMM-yyyy
-                    DateFormat inputFormat = new SimpleDateFormat("yyyy-MM-dd");
-                    DateFormat outputFormat = new SimpleDateFormat("dd MMM yyyy");
-                    String inputDateStr= res.getDate();
-                    Date date = null;
-                    try {
-                        date = inputFormat.parse(inputDateStr);
-                    } catch (ParseException e) {
-                        e.printStackTrace();
-                    }
-                    String outputDateStr = outputFormat.format(date);
-
-                    resNotificationTextView.setText(outputDateStr + " · " + res.getClassName() +
-                            " · " + res.getExpertName());
-                    resNotificationTextView.setSelected(true);
-                }
-                //no reservation
-                else {
-                    resNotificationTextView.setVisibility(View.GONE);
-                }
-            }
-
-            @Override
-            public void onFailure(Call<Reservation> call, Throwable t) {
-                //TODO (kahye) : error handling
-            }
-        });
-    }
-
-    public void click(View view) {
-        Intent confirmResIntent = new Intent(PlaceActivity.this,
-                ConfirmReservationActivity.class);
-
-        Bundle bundle = new Bundle();
-        confirmResIntent.putExtra("_classInfo", selectedClass);
-        confirmResIntent.putExtra("_date", selectedDate);
-        confirmResIntent.putExtras(bundle);
-
-        startActivity(confirmResIntent);
-    }
-
-    @Override
-    public boolean onCreateOptionsMenu(Menu menu) {
-        getMenuInflater().inflate(R.menu.actionbar_actions, menu);
-        return true;
-    }
-
-    @Override
-    public boolean onOptionsItemSelected(MenuItem item) {
-        LoginManager.getInstance().logOut();
-
-        Intent mainIntent = new Intent(PlaceActivity.this, MainActivity.class);
-        startActivity(mainIntent);
-        finish();
-        /*
-        switch (item.getItemId()) {
-            case R.id.action_settings :
-                // TODO (kahye) : process the click event for action_search item.
-                //when we need another actionbar item
-                return true ;
-            default :
-                return super.onOptionsItemSelected(MenuItem ) ;
-        }*/
-        return true;
-    }
-}
+package uncommon.common.activity;

```

Comment:
Please work with Gayeon to fix this view - your git client is causing this file to show the entire file to be in the diff.